Extensions/tweaks.js

```diff
@@ -307,7 +307,8 @@ XKit.extensions.tweaks = new Object({
 				"Default", "default",
 				"Hidden", "hidden",
 				"White", "white",
-				"Old Red", "oldred",
+				"Old Red - theme-dependent", "oldred",
+				"Old Red - always red", "olderredder"
```

Comment:
i'd prefer this formatted/worded slightly differently

```suggestion
				"Old Red (Palette-dependent)", "oldred",
				"Old Red (Always red)", "olderredder",
```